### PR TITLE
feat(buildTheme): add getBoxShadow helper

### PR DESCRIPTION
**Pull request by leko91**
As discussed in the [Slack convo](https://frontify.slack.com/archives/C041H8F0VEU/p1662117361995179)

### CODE REVIEW

File: src/formatters/tailwind/buildTheme.js [hunk 1]
```diff
@@ -62,6 +62,23 @@ const getColors = ({ tokens }) => {
     };
   }, {});
 };
+const getBoxShadow = ({ tokens, dictionary }) => {
+  const matchingTokens = tokens.filter(
+    (token) =>
+      token.attributes.category === "shadow" &&
+      token.attributes.type === "matrix"
+  );
+  let object = {};
```
Comment (jonasbuechel) [inline src/formatters/tailwind/buildTheme.js:71]: Please name that more meaningful, what does it contain?

**Review by jonasbuechel — CHANGES_REQUESTED** [review]
It's solved in an elegant way. I compared all the dist output and it only affected tailwind.config.js and no Figma output 👍 

I found two issues after running `npm run build:debug`

dist/tailwind/tailwind.config.js
```
"boxShadow": {
  "inner-line": "inset 0 0 0 var(--0.063rem) var(--rgba(8, 8, 8, 0.1)))",
  "inner-line-strong": "inset 0 0 0 var(--0.063rem) var(--rgba(8, 8, 8, 0.15)))"
} 
```
I think var() including the value is wrong here

dist/css/tailwind_unpurged.css
Does not contain the expected classes `tw-shadow-inner-line` and `tw-shadow-inner-line-strong`. I gues that's a sequence error of the one above.

**Comment by leko91** [comment]
@jonasbuechel @thisguyscode I fixed the variables in the build, but I cannot get the css classes to generate. Any ideas?

File: src/formatters/tailwind/buildTheme.js [hunk 2]
```diff
@@ -62,6 +62,23 @@ const getColors = ({ tokens }) => {
...
+      token.attributes.category === "shadow" &&
+      token.attributes.type === "matrix"
+  );
+  let boxShadowObject = {};
+  matchingTokens.forEach((token) => {
+    const key = trimHyphens(token.name.replace("shadow", ""));
+    boxShadowObject[key || "DEFAULT"] = `var(--${token.name})`;
+  });
+  Object.assign(boxShadowObject, {
+    "inner-line": `inset 0 0 0 var(--${dictionary.tokens.line["width"].name}) var(--${dictionary.tokens.line["color"].name}))`,
```
Comment (jonasbuechel) [inline src/formatters/tailwind/buildTheme.js:77]:
```suggestion
    "inner-line": `inset 0 0 0 var(--${dictionary.tokens.line["width"].name}) var(--${dictionary.tokens.line["color"].name})`,
```

Superfluous ')' stops it from being integrated into the CSS output.

File: src/formatters/tailwind/buildTheme.js [hunk 3]
```diff
@@ -62,6 +62,23 @@ const getColors = ({ tokens }) => {
...
+    "inner-line-strong": `inset 0 0 0 var(--${dictionary.tokens.line["width"].name}) var(--${dictionary.tokens.line["color-strong"].name}))`,
```
Comment (jonasbuechel) [inline src/formatters/tailwind/buildTheme.js:78]:
```suggestion
    "inner-line-strong": `inset 0 0 0 var(--${dictionary.tokens.line["width"].name}) var(--${dictionary.tokens.line["color-strong"].name})`,
```

Superfluous ')' stops it from being integrated into the CSS output.

**Review by jonasbuechel — CHANGES_REQUESTED** [review]

**Pull request merged**